src/server/getters/get-reporting-fees.ts

```diff
@@ -157,13 +155,13 @@ function formatMarketInfo(marketParticipants: MarketParticipantRows) {
   if (forkedMarket) {
     forkedMarket.crowdsourcers = [];
   }
-  const keyedNonforkedMarkets: {[marketId: string]: NonforkedMarket} = {};
+  const keyedNonforkedMarkets: { [marketId: string]: NonforkedMarket } = {};
   let i: number;
   for (i = 0; i < marketParticipants.initialReporters.length; i++) {
     if (marketParticipants.initialReporters[i].forking) {
       forkedMarket.initialReporter = {
         initialReporterId: marketParticipants.initialReporters[i].initialReporter,
-        needsFork: marketParticipants.initialReporters[i].disavowed ? false : true,
+        needsFork: !marketParticipants.initialReporters[i].disavowed,
```

Comment:
It looks like `initial_reports.disavowed` is an integer, so I think this should be "!!"